src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -102,7 +102,7 @@ class GemUtils {
 		     '--no-user-install',
 		     '--wrappers',
                      '-N',
-                     '--platform=java'
+		     '-l'
```

Comment:
Will you replace this with the long-form version of the option (`--local`) so it's more obvious to future maintainers what this command is doing